Sources/Defaults/DefaultsBridge.swift

```diff
@@ -0,0 +1,185 @@
+import Foundation
+
+public protocol DefaultsBridge {
+	// The type of Value of Key<Value>
```

Comment:
Super nitpick, but end comments with dot. And:
```suggestion
	/// The type of `Value` of `Key<Value>`.
```